_pages/dev/features/asm.md

```diff
@@ -0,0 +1,167 @@
+
+
+```
+
+### Required CMS Data
+
+The ASM angular component is layed out by the CMS and requires a corresponding CMS entry to be displayed in spartacus.
+
+#### Option 1: Initialize with the latest Spartacus sample data
+
+The most straignforward way obtain the ASM CMS component is if you initialize your store with the latest spartacus sample data extension: [spartacussampledataaddon](https://sap.github.io/cloud-commerce-spartacus-storefront-docs/assets/other/spartacussampledataaddon.zip).
+
+#### Option 2: Manual import in impex console.
+
+If you are not starting from scratch with the latest Spartacus sample data, you can import this impex to create the AsmComponent in the CMS and link it to the TopHeaderSlot. You may need to change the name or version of the content catalog to fit your project:
+
+```
+$contentCatalog=electronics-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+
+INSERT_UPDATE CMSFlexComponent;$contentCV[unique=true];uid[unique=true];name;flexType;&componentRef
+;;AsmComponent;asm flex component;AsmComponent;AsmComponent
+
+INSERT_UPDATE ContentSlot;$contentCV[unique=true];uid[unique=true];active;cmsComponents(&componentRef)[mode=append]
+;;TopHeaderSlot;true;AsmComponent
+
+```
+
+### CORS Configuration
+
+The `assistedservicewebservices` extension requires CORS configuration, which is possible since SAP Commerce Cloud version 1905.5. In your `local.properties` file, you need to customize the "allowed origins" property for `assistedservicewebservices`, which is shown here with its default value:
+
+```
+corsfilter.assistedservicewebservices.allowedOrigins=http://localhost:4200 https://localhost:4200
+```
+
+For development purposes, the value can be a wildcard:
+
+```
+corsfilter.assistedservicewebservices.allowedOrigins=*
+```
+
+Bear in mind this wildcard configuration is flexible for development environments but it is unsecure. A more restrictive configuration is required for production use.
+
+## Invoke the Asm UI in the storefront.
+
+To invoke the ASM UI in the storefront, add the `?asm=true` suffix to the url.
+For example, with the sample store, you can invoke the ASM UI on the home page with this url
+
+```
+https://{hostname}/electronics-spa/en/USD/?asm=true
+```
+
+## Enabling/Disabling the ASM UI in Spartacus
+
+Wether you initialized with the latest Spartacus sample data or manually imported the impex above to import the AsmComponent, the Asm UI will be enabled by default.
+
+The `visibility` attribute of AsmComponent in the CMS dictates wether the ASM UI is enabled or not in the storefront. When AsmComponent is not visible, it will be disabled and completely omitted in the page data returned by OCC to be rendered by Spartacus.
+
+An "invisible" AsmComponent will therefore not display even if invoked with `?asm=true`.
+
+## Spartacus ASM aware development guidelines
```

Comment:
Should the section `Spartacus ASM aware development guidelines` be somewhere else, on another page?